src/components/Modals/ConvertToLatestModal.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import toast from 'react-hot-toast';
+
+import { ConfirmModal } from './ConfirmModal';
+
+import { EditorService, SpecificationService } from '../../services';
+import state from '../../state';
+
+export const ConvertToLatestModal: React.FunctionComponent = () => {
+  const [show, setShow] = useState(false);
+  const specState = state.useSpecState();
+  const shouldOpenConvertModal = specState.shouldOpenConvertModal.get();
+
+  const latestVersion = SpecificationService.getLastVersion();
+
+  useEffect(() => {
+    shouldOpenConvertModal && setShow(true);
+  }, [shouldOpenConvertModal]);
+
+  const onSubmit = () => {
+    async function convert() {
+      try {
+        await EditorService.convertSpec();
+      } catch (e) {
+        throw e;
+      } finally {
+        specState.shouldOpenConvertModal.set(false);
+        setShow(false);
+      }
+    }
+
+    toast.promise(convert(), {
+      loading: 'Converting...',
+      success: (
+        <div>
+          <span className="block text-bold">
+            Document succesfully converted!
+          </span>
+        </div>
+      ),
+      error: (
+        <div>
+          <span className="block text-bold text-red-400">
+            Failed to convert document.
+          </span>
+        </div>
+      ),
+    });
+  };
+
+  return (
+    <ConfirmModal
+      title="Convert AsyncAPI document to latest version"
+      confirmText={`Convert to ${latestVersion}`}
+      confirmDisabled={false}
+      show={show}
+      onSubmit={onSubmit}
+    >
+      <div className="flex flex-col content-center justify-center text-center">
+        <p>
+          Your document is using an old version of AsyncAPI. The Studio only supports 2.0.0 and above. Convert your document to latest ({latestVersion}) version to continue.
+        </p>
+        <div className="mt-8">
+          <a
+            className="underline"
+            href={`https://www.asyncapi.com/blog/release-notes-${latestVersion}`}
+            target="_blank"
+            rel="nofollow noopener noreferrer"
+          >
+            See the release notes
+          </a>
+        </div>
```

Comment:
Should we not show the release notes for each version inbetween? i.e. if you have a document with version `2.0.0` and we currently are in `2.2.0` it should show release notes for `2.1.0` and `2.2.0` 🤔

Reply:
It only shows release notes for latest, but it's a good idea to show all between current version and latest. I will propagate it. Thanks!

Reply:
Mock for `2.0.0` version (and also for older than `2.0.0` - we don't have release notes for `2.0.0` so I won't show any link):

![image](https://user-images.githubusercontent.com/20404945/136058805-863b3e0f-dee6-4bcb-9b4a-97035a6295b4.png)

Mock for `2.1.0` version:

![image](https://user-images.githubusercontent.com/20404945/136058902-7c6e2c27-b75f-4be8-95fb-33227ae5703e.png)

Reply:
This reminds me based on last PR - https://github.com/asyncapi/studio/pull/100

Should the user be able to select another version to convert to then the latest version? 🤔

Reply:
I can do this :)

Reply:
But... Between 2.0.0 and 2.2.0 there are only new things that help more than hinder. We could implement something like this when we will release 3.0.0, what do you think? I can't think of a use case if someone wanted a version smaller than 2.2.0 (atm).

Reply:
It makes sense yea, I cant think of a use-case either, however when you have the option in the other functionality to decide between versions, I would assume the same behavior where consistent here as well 🙂

Reply:
Ok, I will do it tomorrow :) Thanks for suggestions!